packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -134,8 +134,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     // (undocumented)
     readonly options: Record<string | number, any>;
     orderSequentially<T>(callback: () => T): T;
-    // (undocumented)
-    process(messageArg: ISequencedDocumentMessage, local: boolean): void;
+    process({ ...messageCopy }: ISequencedDocumentMessage, local: boolean): void;
```

Comment:
Seems like this change should have a changeset if it changes publicly exposed APIs.

Reply:
It doesn't change the type, just the "spelling" (the implementation destructures the argument to do a shallow clone)